google-home-community.groovy

```diff
@@ -432,27 +434,27 @@ private deviceTraitPreferences_Brightness(deviceTrait) {
 
 @SuppressWarnings('UnusedPrivateMethod')
 def deviceTraitPreferences_CameraStream(deviceTrait) {
-    googleCameraStreamSupportedProtocols = [
-        "progressive_mp4":         "Progressive MP4",
-        "hls":                     "HLS",
-        "dash":                    "Dash",
-        "smooth_stream":           "Smooth Stream",
-    ]
     section("Stream Camera") {
         input(
-            name: "${deviceTrait.name}.cameraStreamAttribute",
+            name: "${deviceTrait.name}.cameraStreamURLAttribute",
```

Comment:
Previous implementation only permitted a single camera protocol since it was based in the app.  Changes moved this selection to the driver to allow a multitude of camera protocol types which is breaking backwards compatibility in this case.

Reply:
Need to break backward compatibility to move the URLs to the driver level.